Lambda-School-Labs/CS10-quizzer
# Brandon/quiz urls

# Description

Can now load a `Quiz` depending on the URL
Can answer `Questions` and submit `Quiz` to receive a score that is saved in the DB

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- Create a Teacher, Class, Quiz (for that class), Questions (for that quiz), Choices/Answrs (for that question), Student (add to the class)
- Navigate to `http://localhost:3000/<<quiz ID>>/<<class ID>>/<<student ID>>` and take the quiz
- When finished you should get an unstyled page letting you know your score on that quiz
- Check database for confirmation in `QuizScores` table

# Checklist:
